### PR TITLE
changelog updated for the release 1.15.0

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
### What does this PR do?
Updated changelog for the release 1.15.0

### What issues does this PR fix or reference?
NA

### Is it tested? How?
NA

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+### 1.15.0
```
Comment (datho7561) [inline CHANGELOG.md:1]: I think that these two commits don't have a corresponding entry in the README:
- https://github.com/redhat-developer/yaml-language-server/commit/02d69fe2908e68184ed130b936b4e0107122620e
- https://github.com/redhat-developer/yaml-language-server/commit/2f2683b105229dc79734dd84149450a5d43bb4e4

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (msivasubramaniaan) [reply CHANGELOG.md:1]: @datho7561 https://github.com/redhat-developer/yaml-language-server/commit/02d69fe2908e68184ed130b936b4e0107122620e was closed without merge the [PR](https://github.com/redhat-developer/yaml-language-server/pull/923) and https://github.com/redhat-developer/yaml-language-server/commit/2f2683b105229dc79734dd84149450a5d43bb4e4 was closed without merge the [PR](https://github.com/redhat-developer/yaml-language-server/pull/924). So I haven't made an entry

Comment (msivasubramaniaan) [reply CHANGELOG.md:1]: aww, sorry missed the logs. Done. Please have a look

**Review by datho7561 — APPROVED** [review]
Looks good to me!

**Review by rgrunber — APPROVED** [review]

**Pull request merged**